### PR TITLE
Feature/fw configuration

### PR DESCRIPTION
**Pull request by cassiaabarbosa**
## Motivation
#67 
#58 

## Modifications
- Added color style option.
- Added FWConfiguration for configuring the form.
- Added protocol defining FWFormGenerator behaviour.

## Result
Users can now better configure a form.
**We still need to determine which color apply to which element** 

##  Checklist
 **This PR is**
* [x] In accordance with our coding principles.
* [x] Implementing tests wherever needed and possible.
* [x] Free of commented code.
* [x] Documented.
* [x] Closing an Issue or related to an Issue
* [x] Approved in the CI

### CODE REVIEW

File: Sources/Formworks/Design System/FWStyle.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+/// The form's color style.
+public enum FWStyle {
+	/// The default light color style of a form.
+	case light
+	
+	/// The default dark color style of a form.
+	case dark
+	
+	/// A custom color style of a form.
+	case custom(FWStyleSpecification)
```
Comment (VicFalcetta) [inline Sources/Formworks/Design System/FWStyle.swift:19]: How will be the custom color scheme be implemented? I really forgot during the day on what we discussed about the custom colors.

File: Sources/Formworks/Design System/FWStyle.swift [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import Foundation
+
+/// The form's color style.
+public enum FWStyle {
+	/// The default light color style of a form.
+	case light
+	
+	/// The default dark color style of a form.
+	case dark
```
Comment (VicFalcetta) [inline Sources/Formworks/Design System/FWStyle.swift:16]: Nice to know that both modes are in theory implemented. Due to this, we need to put back on the style documentation the dark mode color codes.

**Review by VicFalcetta — COMMENTED** [review]
Since during my review there were still some changes I'm going to wait a bit until I'll approve.

**Review by VicFalcetta — DISMISSED** [review]

File: Sources/Formworks/Design System/FWStyle.swift (hunk 2 above)
Comment (csfar) [reply Sources/Formworks/Design System/FWStyle.swift:16]: We still need to add some more colors since we've added new customisation options.

**Review by Galdineris — APPROVED** [review]

**Review by VicFalcetta — APPROVED** [review]

**Pull request merged**